tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -193,5 +198,65 @@ def build(ctx: click.Context) -> bool:
         )
         for connector in ctx.obj["selected_connectors"]
     ]
-    anyio.run(run_connectors_build_pipelines, connectors_contexts, ctx.obj["concurrency"])
+    anyio.run(run_connectors_pipelines, connectors_contexts, run_connector_build_pipeline, "Build Pipeline", ctx.obj["concurrency"])
+
+    return True
+
+
+@connectors.command(cls=DaggerPipelineCommand, help="Publish all images for the selected connectors.")
+@click.option("--pre-release/--main-release", help="Use this flag if you want to publish pre-release images.", default=True, type=bool)
+@click.option(
+    "--spec-cache-service-account-key",
+    help="The service account key to upload files to the GCS bucket hosting spec cache.",
+    type=click.STRING,
+    required=True,
+    envvar="SPEC_CACHE_SERVICE_ACCOUNT_KEY",
+)
+@click.option(
+    "--spec-cache-bucket-name",
+    help="The name of the GCS bucket where specs will be cached.",
+    type=click.STRING,
+    required=True,
+    envvar="SPEC_CACHE_BUCKET_NAME",
+)
+@click.pass_context
+def publish(ctx: click.Context, pre_release: bool, spec_cache_service_account_key: str, spec_cache_bucket_name: str):
+    if ctx.obj["is_local"]:
+        click.confirm(
+            "Publishing from a local environment is not recommend and require to be logged in to Airbyte's DockerHub registry, do you want to continue?",
```

Comment:
Other env variable likes `SPEC_CACHE_BUCKET_NAME` and `SPEC_CACHE_SERVICE_ACCOUNT_KEY` are required, but the CLI will fail beforehand if these are not set.